libs/dh/admin/feature-invite-user-modal/src/lib/dh-invite-user-modal.component.scss

```diff
@@ -53,4 +53,23 @@
     display: block;
     margin-bottom: var(--watt-space-ml);
   }
+
+  .info-section {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    margin-bottom: var(--watt-space-m);
+  }
+
+  .info-icon {
+    margin-right: var(--watt-space-s);
+  }
+
+  .info-icon .mat-icon {
+    color: var(--watt-color-neutral-grey-600) !important;
```

Comment:
nitpick: instead of changing the color of an icon, let's use the `state` Input to set a specific state. Various states can be seen at the bottom of this page. https://main--61765fc47451ff003afe62ff.chromatic.com/?path=/story/foundations-icons--icons